modules/configuring-aws-iam-for-cross-account-access.adoc

```diff
@@ -0,0 +1,78 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-10-19
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-aws-iam-for-cross-account-access_{context}"]
+= Configuring AWS IAM for cross-account access
+
+You can configure {aws-short} Identity Access Management (IAM) for cross-account access.
+
+.Prerequisites
+* You have administrator permissions.
+
```

Comment:
The target role had been created and has permissions to put data into cloudwatch or s3.